docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,666 @@
+ - `c :#= p` (coercing mono-direction): connects all p members to c; regardless of alignment
+ - `c :<= p` (aligned-direction); connects all aligned (non-flipped) c members from p
+ - `c :>= p` (flipped-direction); connects all flipped p members from c
+ - `c :<>= p` (bi-direction operator); connects all aligned c members from p; all flipped p members from c
+
+You may be seeing these random symbols in the operator and going "what the heck are these?!?". Well, it turns out that the characters are consistent between operators and self-describe the semantics of each operator:
+ - `:` always indicates the consumer, or left-hand-side, of the operator.
+ - `=` always indicates the producer, or right-hand-side, of the operator.
+   - Hence, `c := p` connects a consumer (`c`) and a producer (`p`).
+ - `<` always indicates that some members will be driven producer-to-consumer, or right-to-left.
+   - Hence, `c :<= p` drives members in producer (`p`) to members in consumer (`c`).
+ - `>` always indicates that some signals will be driven consumer-to-producer, or left-to-right.
+   - Hence, `c :>= p` drives members in consumer (`c`) to members producer (`p`).
+   - Hence, `c :<>= p` both drives members from `p` to `c` and from `c` to `p`.
+ - `#` always indicates to ignore member alignment and to drive producer-to-consumer.
+   - Hence, `c :#= p` always drives members from `p` to `c` ignoring direction.
+
+
+## Alignment: Flipped vs Aligned
+
+A member's alignment is a relative property: a member is aligned/flipped relative to another member of the same component or Chisel type. Hence, one must always say whether a member is flipped/aligned *with respect to (w.r.t)* another member of that type (parent, sibling, child etc.).
+
+We use the following example of a non-nested bundle `Parent` to let us state all of the alignment relationships between members of `p`.
+
+```scala mdoc:silent
+import chisel3._
+class Parent extends Bundle {
+  val alignedChild = UInt(32.W)
+  val flippedChild = Flipped(UInt(32.W))
+}
+class MyModule0 extends Module {
+  val p = Wire(new Parent)
+}
+```
+
+First, every member is always aligned with themselves:
+ - `p` is aligned w.r.t `p`
+ - `p.alignedChild` is aligned w.r.t `p.alignedChild`
+ - `p.flippedChild` is aligned w.r.t `p.flippedChild`
+
+Next, we list all parent/child relationships. Because the `flippedChild` field is `Flipped`, it changes its aligment relative to its parent. 
+ - `p` is aligned w.r.t `p.alignedChild`
+ - `p` is flipped w.r.t `p.flippedChild`
+
+Finally, we can list all sibling relationships:
+ - `p.alignedChild` is flipped w.r.t `p.flippedChild`
+
+The next example has a nested bundle `GrandParent` who instantiates an aligned `Parent` field and flipped `Parent` field.
+
+```scala mdoc:silent
+import chisel3._
+class GrandParent extends Bundle {
+  val alignedParent = new Parent()
+  val flippedParent = Flipped(new Parent())
```

Comment:
```suggestion
  val alignedParent = new Parent
  val flippedParent = Flipped(new Parent)
```
Generally there should be parentheses if the class takes no arguments, and we should also try to be consistent (above and below you seems to do both equally).

Reply:
Yeah, it's one of my many flaws is a blindness to this particular syntactic issue... Thanks!